docs/projects/version-control/converting/migrating-variables.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Migrating variables to Git
+description: Variables needs to be manually migrated to Git for existing projects
+position: 110
+hideInThisSection: true
+---
+
+Support for storing non-senstive variables in Git will be available to everyone in an upcoming version of Octopus. All newly configured Git projects will automatically have variables migrated when converting the project to Git - for existing Git projects, you will need to migrate these manually.
+
+
+:::hint
+We recommend migrating your variables to Git as soon as practical. We will continue to support Git projects with un-migrated variables for now, but we will remove support for this at some point in the future.
+:::
+
+This document covers the process of migrating variables to Git.
+
+## Migrating variables to Git
+
+You will need to migrate the variables for each existing Git project separately.
+
+### Preparing to migrate
+
+#### Check that all secrets are marked as sensitive
+
+All non-sensitive values will be written to your Git repository in plain text during the migration. Before migrating, make sure that you do not have any secret values that are not stored as sensitive variables.
+
+
+:::warning
+If you can view a value on the on the variables page (and it's not shown as a password field) or retrieve the value it from the API, it's not marked as sensitive and will be written to the `variables.ocl` file.
+:::
+
+#### Coordinate with your team
+
+The migration process will only write the variables to a single branch. After migrating you will need to merge this change to your other branches to continue using those branches in Octopus. If a branch does not have a `variables.ocl` file after the migration, you will get an error when trying to read.
```

Comment:
```suggestion
The migration process will only write the variables to a single branch. After migrating, you will need to merge this change to your other branches to continue using those branches in Octopus. If a branch does not have a `variables.ocl` file after the migration, you will get an error when trying to read.
```

Reply:
Can borrow some words that I wrote for the warning dialog:
> Any existing Git tags or branches that do not have the migration commit will not be able to create releases, run runbooks, edit deployment processes, update variables etc.

Reply:
I've reworked this to remove the vague "you will get an error", and add some examples of when it might fail. I don't want to call out specific error messages, because they're likely to change, there are a few possible error messages, and a few possible causes of those error messages.

This is what I have now:

```
### Coordinate with your team

After you have run the migration, you will be unable to read variables from any Git reference (branch, tag, or commit) that does not contain a valid `variables.ocl` file. This will cause failues when viewing and editing variables, creating releases, running runbooks using those references until they're updated.

As the migration process only writes the variables to a single branch, you will need to merge that migration branch with your other branches to continue using them in Octopus. If this may disrupt your team, plan ahead and find a suitable time to migrate and merge.
```

Reply:
LGTM. You _could_ remove the As and start with `The` e.g.:

> The migration process only writes the variables to a single branch. To continue using the variables in any other branches, you will need to merge that migration branch with those branches. If this may disrupt your team, plan ahead and find a suitable time to migrate and merge.